Composer/packages/client/src/components/Notifications/NotificationCard.tsx

```diff
@@ -112,13 +113,20 @@ const cardDescription = css`
   word-break: break-word;
 `;
 
-const linkButton = css`
-  color: #0078d4;
-  float: right;
-  font-size: 12px;
-  height: auto;
-  margin: 4px 0 4px 8px;
-`;
+const linkButton = {
+  root: {
+    padding: '0',
```

Comment:
`padding` and `border` should be `0px`. Also, why are you setting the border styles? `ActionButton`s don't have borders by default.

Reply:
0 is equivalent to 0 in any unit; there's no need to specify pixels there. And there was a 1 pixel border there that this eliminates, just to align to the designs better.